styles.css

```diff
@@ -1,504 +1,1092 @@
-body {
-  background: #e5e5e5;
-  margin: 0;
-  padding: 0;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-}
+@media only screen and (max-width: 767.9px) {
+  body {
+    background: #e5e5e5;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
 
-.header {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: space-between;
-  padding-top: 25px;
-  width: 375px;
-}
+  .header {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 25px;
+    padding-bottom: 25px;
+    width: 375px;
+    background-color: #fff;
+  }
 
-.union {
-  padding-right: 20px;
-}
+  .union {
+    padding-right: 20px;
+  }
 
-#logo {
-  text-decoration: none;
-  width: 106px;
-  height: 20px;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 25px;
-  line-height: 20px;
+  .top_navbar {
+    display: none;
+  }
 
-  /* identical to box height, or 111% */
+  #logo {
+    text-decoration: none;
+    width: 106px;
+    height: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 25px;
+    line-height: 20px;
 
-  display: flex;
-  align-items: center;
-  text-align: center;
-  padding-left: 17px;
+    /* identical to box height, or 111% */
 
-  /* Primary/B400 */
+    display: flex;
+    align-items: center;
+    text-align: center;
+    padding-left: 17px;
 
-  color: #6070ff;
-}
+    /* Primary/B400 */
 
-.main-section {
-  display: flex;
-  flex-direction: column;
-  background-image: url(./images/mobie.png);
-  width: 375px;
-  height: 616.15px;
-  left: 0;
-  top: 75px;
-  margin-bottom: 20px;
-  margin-top: 30px;
-}
+    color: #6070ff;
+  }
 
-.primary-text {
-  width: 317px;
-  height: 150px;
-  margin-top: 100px;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 800;
-  font-size: 40px;
-  line-height: 52px;
-  display: flex;
-  align-items: center;
-  padding-left: 26px;
-  color: #172b4d;
-}
+  .main-section {
+    display: flex;
+    flex-direction: column;
+    background-image: url(./images/mobie.png);
+    width: 375px;
```

Comment:
- [OPTIONAL] Great job on implementing the Figma design for the desktop version perfectly well, 👍 💯 ... 
- - I notice that on-screen sizes between `376px` and `769px`, the `main` section of your page does not span fully to the left and right of the screen and this is due to the specific width you gave, Please I'll suggest you remove this and use `Relative width units` like `width: 100%;`, Your portfolio will look great on all screens. Thanks 😄 
- SCREENSHOT FOR REFERENCE:
![image](https://user-images.githubusercontent.com/107632780/229273930-eca25376-2d79-4d25-91b4-375b6a80e3ad.png)